sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -136,3 +136,13 @@ export const OFFERING_PROGRAM_YEAR_MISMATCH = "OFFERING_PROGRAM_YEAR_MISMATCH";
  */
 export const OFFERING_DOES_NOT_BELONG_TO_LOCATION =
   "OFFERING_DOES_NOT_BELONG_TO_LOCATION";
+/**
+ * Student not authorized for the offering details summary.
+ */
+export const STUDENT_UNAUTHORIZED_FOR_OFFERING =
```

Comment:
As a suggestion I would recommend to avoid `STUDENT`. Something like `NOT_AUTHORIZED_FOR_OFFERING` or  `NOT_AUTHORIZED_FOR_OFFERING_PURPOSE`